playbooks/complex/demo_file_access.yml

```diff
@@ -1,12 +1,14 @@
 ---
 - hosts: localhost
+  vars_files:
+    - ../../group_vars/all
   tasks:
     - name: get disks
       virtualDisk:
        login:
-          unityIP: "{{ host }}"
-          username: "{{ username }}"
-          password: "{{ password }}"
+          unityIP: "{{ unity_host }}"
```

Comment:
As youo mentioned: "In #41 and #44 we remove unity_host group var. Let's change it to unity_ip_source or unity_ip_dest"